applications/DEMApplication/python_scripts/sphere_strategy.py

```diff
@@ -1,3 +1,4 @@
+from pickle import FALSE, TRUE
```

Comment:
Is this import necessary?

Reply:
This import was automatically added when I was trying to use FALSE and TRUE. It is not necessary now and has been removed.